### PR TITLE
[WIP] harmonizing the structure of Jasmine tests

### PR DESCRIPTION
**Pull request by wKoza**
For harmonization, we can add more types in our templates.

### CODE REVIEW

File: packages/angular-cli/blueprints/ng2/files/__path__/app/app.component.spec.ts [hunk 1]
```diff
@@ -13,21 +13,21 @@ describe('App: <%= jsComponentName %>', () => {
   });
 
   it('should create the app', async(() => {
-    let fixture = TestBed.createComponent(AppComponent);
-    let app = fixture.debugElement.componentInstance;
+    let fixture:ComponentFixture<AppComponent> = TestBed.createComponent(AppComponent);
```
Comment (grizzm0) [inline packages/angular-cli/blueprints/ng2/files/__path__/app/app.component.spec.ts]: There should be a space here and on all following types or the linting will fail.

Comment (wKoza) [inline packages/angular-cli/blueprints/ng2/files/__path__/app/app.component.spec.ts]: right! ty

File: packages/@angular/cli/blueprints/pipe/files/__path__/__name__.pipe.spec.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { <%= classifiedModuleName %>Pipe } from './<%= dasherizedModuleName %>.p
 
 describe('<%= classifiedModuleName %>Pipe', () => {
   it('create an instance', () => {
-    const pipe = new <%= classifiedModuleName %>Pipe();
+    let pipe: <%= classifiedModuleName %>Pipe = new <%= classifiedModuleName %>Pipe();
```
Comment (grizzm0) [inline packages/@angular/cli/blueprints/pipe/files/__path__/__name__.pipe.spec.ts]: This should be left as a const

Comment (wKoza) [inline packages/@angular/cli/blueprints/pipe/files/__path__/__name__.pipe.spec.ts]: ok for const

File: packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts [hunk 3]
```diff
@@ -18,21 +18,21 @@ describe('AppComponent', () => {
   });
 
   it('should create the app', async(() => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.debugElement.componentInstance;
+    const fixture: ComponentFixture<AppComponent> = TestBed.createComponent(AppComponent);
```
Comment (clydin) [inline packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts]: The component fixture type can be inferred.  There is no need to specify.

File: packages/@angular/cli/blueprints/pipe/files/__path__/__name__.pipe.spec.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { <%= classifiedModuleName %>Pipe } from './<%= dasherizedModuleName %>.p
 
 describe('<%= classifiedModuleName %>Pipe', () => {
   it('create an instance', () => {
-    const pipe = new <%= classifiedModuleName %>Pipe();
+    const pipe: <%= classifiedModuleName %>Pipe = new <%= classifiedModuleName %>Pipe();
```
Comment (clydin) [inline packages/@angular/cli/blueprints/pipe/files/__path__/__name__.pipe.spec.ts]: The type can be inferred here as well.

File: packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts (hunk 3 above)
Comment (wKoza) [inline packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts]: For harmonization with [component template](https://github.com/angular/angular-cli/blob/master/packages/@angular/cli/blueprints/component/files/__path__/__name__.component.spec.ts#L10)

Comment (clydin) [inline packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts]: The structures of the two files are different.  I think harmonizing the structures of the two component specs might be something to consider.
Either way, typescript is quite good at inferring types.  In fact, TS 2.1+ (with noImplicitAny enabled) can actually infer the type shown in the linked component template.

**Comment by filipesilva** [comment]
I'm sorry but I'm not merging this PR. 

At the moment we're scrambling to put the finishing touches on rc.0 and this PR can be considered somewhat of a breaking change since it alters output users see and thus would need to go in prior to rc.0.

It also doesn't fix anything per se (it's just a matter of style), so there's no pressing reason to merge.

I appreciate the effort you put in, but this was bad timing.

**Pull request closed without merging**